### PR TITLE
[DR-3119] Use new azure TF'ed secrets

### PR DESCRIPTION
**Pull request by nmalfroy**
There's a new application/set of secrets that are now Terraformed.  We should use those now

### CODE REVIEW

File: perf/datarepo/create-secret-manager-secret.yaml [hunk 1]
```diff
@@ -17,17 +17,17 @@ secrets:
   - secretName: azure-perf
     vals:
       - kubeSecretKey: applicationsecret
-        path: secret/dsde/datarepo/perf/helm-azure
-        vaultKey: applicationsecret
+        path: secret/dsde/datarepo/dev/azure-application-secrets
+        vaultKey: client-secret
```
Comment (snf2ye) [inline perf/datarepo/create-secret-manager-secret.yaml:21]: Same here, for perf?

File: integration/integration-2/create-secret-manager-secret.yaml [hunk 2]
```diff
@@ -17,17 +17,17 @@ secrets:
   - secretName: azure-integration-2
     vals:
       - kubeSecretKey: applicationsecret
-        path: secret/dsde/datarepo/integration/helm-azure-integration
-        vaultKey: applicationsecret
+        path: secret/dsde/datarepo/dev/azure-application-secrets
+        vaultKey: client-secret
```
[the remark this answers is not in the document]
Comment (samanehsan) [reply integration/integration-2/create-secret-manager-secret.yaml:21]: I'm wondering the same thing 🤔

[the remark this answers is not in the document]
Comment (nmalfroy) [reply integration/integration-2/create-secret-manager-secret.yaml:21]: There isn't.  So the challenge is that the terraform creates resources for the different environments as part of the secret creation but it felt like overkill given that we use a lot of dev resources in integration and perf already.  That being said, I'm happy to create those in the TF

[the remark this answers is not in the document]
Comment (snf2ye) [reply integration/integration-2/create-secret-manager-secret.yaml:21]: I think it's fine without it, I just wanted to make sure it was on purpose.

**Review by snf2ye — APPROVED** [review]

**Review by samanehsan — APPROVED** [review]

**Comment by samanehsan** [comment]
Could you update the Data Repo set up docs to include the new variables we need to have set? https://github.com/DataBiosphere/jade-data-repo/blob/develop/docs/jade-getting-started.md#1-build-jade-data-repo

**Comment by nmalfroy** [comment]
I updated the values for integration and perf with the new resources and secrets that I introduced here:
https://github.com/broadinstitute/terraform-ap-deployments/pull/1143

**Comment by nmalfroy** [comment]
Ah, yeah, I'll update the render-configs script too! (Though FWIW, for local dev, most folks probably don't need to set the synapse initialization variable...)

**Pull request merged**